tests/integration/targets/postgresql_subscription/defaults/main.yml

```
@@ -1,6 +1,6 @@
 pg_user: postgres
 db_default: postgres
-primary_port: 5433
+primary_port: 5431
```

```suggestion
primary_port: 5431
```
if it works, we could try to remove it from here as it's already defined in `tests/integration/targets/setup_postgresql_replication/defaults/main.yml` (i'm not 100% sure but it should work w/o having it here as well as in the same file of postgresql_info target)

---

no need to remove it now but we could try at the end, after CI is green

---

ok, trying this now